Header header
int16 servo0
int16 servo1
int16 servo2
int16 servo3
int16 servo4
int16 servo5
int16 servo6
int16 servo7
int16 servo8
int16 servo9
int16 servo10
int16 servo11
int16 servo12
int16 servo13
int16 servo14
int16 servo15
int16 servo16
int16 servo17